galaxy.yml

```diff
@@ -0,0 +1,15 @@
+namespace: middleware_automation
+name: redhat-csp-download
```

Comment:
All names need to use `_` instead of `-`

Reply:
@sabre1041 excellent point! @hcherukuri we should at least add redhat, rhel, rhn, subscription_manager as tags.